File: easy/gotham-city/GothamCity.c
#include <stdio.h>

int main() {
    int criminals;
    scanf("%d", &criminals);
    
    if (criminals < 5) puts("I got this!");
    else if (criminals < 11) puts("Help me Batman");
    else puts("Good Luck out there!");
    
    return 0;
}

File: easy/gotham-city/GothamCity.java
public class GothamCity {
    public static void main(String[] args) {
        System.out.print(java.text.MessageFormat.format(
            "{0,choice,0<I got this!|4<Help me Batman|10<Good Luck out there!}",
            new java.util.Scanner(System.in).nextInt())
        );
    }
}
